dist/profile/manifests/jiraveg.pp

```diff
@@ -0,0 +1,32 @@
+# profile for jira
+class profile::jiraveg (
+  $jira_javahome,
+  $jira_version
+){
+  $pkgs = ['openjdk-7-jre-headless', 'facter']
+  package { $pkgs :
+    ensure    => present
+  } ->
+  class { 'jira':
+    version     => $jira_version,
+    javahome    => $jira_javahome,
+    installdir  => '/opt/jira',
+    homedir     => '/srv/jira_home',
+    user        => 'jira_user',
+    group       => 'jira',
+
+    # Database Settings
+    db          => 'postgresql',
```

Comment:
All of these settings should come from hiera directly. If you add some defaults into hieradata/common.yaml they should "just work".

See http://stackoverflow.com/questions/19136413/how-to-write-class-parameters-into-hiera